### PR TITLE
Feature/note status clear

### PR DESCRIPTION
**Pull request by CalebAllard**
# Description
 Added style to Private and Public buttons on notes. Re-added Project Manager to Project view team list. 

## Type of change
This is a temporary fix to force a page refresh upon creation of new note until a proper rework can be implemented based on an error/loop/semi-stateless implementation needing a rework in the application. As that rework is completed this 3 line patch code should no longer be needed.
```
setTimeout(function () {
          window.location.href = `/project/${projectId}`;
        }, 250);
```
src/components/Project/NoteEditor/index.jsx

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status
- [x] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/Project/NoteEditor/NoteEditor.module.scss [hunk 1]
```diff
@@ -159,3 +159,13 @@ h2 {
     }
   }
 }
+.notewraperPrivit {
```
Comment (lawsonarichard) [inline src/components/Project/NoteEditor/NoteEditor.module.scss]: change to` .notewrapperPrivate`

File: src/components/Project/NoteFeed/Note/Notes.module.scss [hunk 2]
```diff
@@ -139,3 +139,27 @@ section:not(:last-child) {
 .deleteButton {
   margin: 0 0.5rem;
 }
+
+.notewraperPublic {
```
Comment (lawsonarichard) [inline src/components/Project/NoteFeed/Note/Notes.module.scss]: change to
`.notewrapperPublic`

File: src/components/Project/NoteFeed/Note/Notes.module.scss [hunk 3]
```diff
@@ -139,3 +139,27 @@ section:not(:last-child) {
...
+  border: solid 1px rgb(245, 73, 135);
+  border-radius: 5px;
+  &:hover {
+    cursor: pointer;
+  }
+  width: 100px;
+  text-align: center;
+  color: rgb(245, 73, 135);
+}
+.notewraperPrivit {
```
Comment (lawsonarichard) [inline src/components/Project/NoteFeed/Note/Notes.module.scss]: change to `.notewrapperPrivate`

File: src/components/Project/NoteFeed/Note/index.jsx [hunk 4]
```diff
@@ -22,6 +22,8 @@ import {
   expanded,
   miniAvatarContainer,
   collapsed,
+  notewraperPrivit,
```
Comment (lawsonarichard) [inline src/components/Project/NoteFeed/Note/index.jsx]: change to 
`notewrapperPrivate`

File: src/components/Project/NoteFeed/Note/index.jsx [hunk 5]
```diff
@@ -22,6 +22,8 @@ import {
   expanded,
   miniAvatarContainer,
   collapsed,
+  notewraperPrivit,
+  notewraperPublic,
```
Comment (lawsonarichard) [inline src/components/Project/NoteFeed/Note/index.jsx]: change to `.notewrapperPublic`

File: src/components/Project/NoteFeed/Note/index.jsx [hunk 6]
```diff
@@ -67,7 +69,9 @@ const Note = ({ note, user, projectManagers, projectId }) => {
               starDimension="20px"
               starSpacing=".5px"
             />
-            {note.privateNote ? "Private Note" : "Public Note"}
+            {note.privateNote ?
+              <div onClick={() => setIsEditing(true)} class={notewraperPrivit}>Private Note</div>
```
Comment (lawsonarichard) [inline src/components/Project/NoteFeed/Note/index.jsx]: change to 
`class={notewrapperPrivate}`

File: src/components/Project/NoteFeed/Note/index.jsx [hunk 7]
```diff
@@ -67,7 +69,9 @@ const Note = ({ note, user, projectManagers, projectId }) => {
               starDimension="20px"
               starSpacing=".5px"
             />
-            {note.privateNote ? "Private Note" : "Public Note"}
+            {note.privateNote ?
+              <div onClick={() => setIsEditing(true)} class={notewraperPrivit}>Private Note</div>
+              : <div onClick={() => setIsEditing(true)} class={notewraperPublic}>Public Note</div>}
```
Comment (lawsonarichard) [inline src/components/Project/NoteFeed/Note/index.jsx]: change to
`class={notewrapperPublic}`

**Review by lawsonarichard — CHANGES_REQUESTED** [review]
Make the changes as following

**Review by lawsonarichard — APPROVED** [review]

**Review by lawsonarichard — APPROVED** [review]
pulled down, tested functions for all components on project single view page

File: src/components/Project/NoteEditor/index.jsx [hunk 8]
```diff
@@ -86,10 +86,13 @@ const NoteEditor = ({ projectManagers, user, projectId, note, setIsEditing }) =>
 				});
 				resetForm();
 				executeQuery();
-				// Temp fix to re-render page, will reload the whole page to populate note after creation.
+				/*START OF FIX
```
Comment (Memitaru) [inline src/components/Project/NoteEditor/index.jsx:89]: Thanks for the note and documentation on the PR. Very helpful. 👍

**Review by Memitaru — APPROVED** [review]
Works for me, looks great 👍

**Pull request merged**